### PR TITLE
HPCC-15655: Allow port number and direct host for mysql server

### PR DESCRIPTION
**Pull request by gaojx**

### CODE REVIEW

**Comment by gaojx** [comment]
@wangkx please code review

File: initfiles/componentfiles/configxml/mysqlserver.xsd [hunk 1]
```diff
@@ -116,12 +116,19 @@
...
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
+                </xs:annotation> 
+            </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
```
Comment (wangkx) [inline initfiles/componentfiles/configxml/mysqlserver.xsd]: @gaojx have you tested this? You may also need to upgrade the xsl files (esp_service.xsl, esp_service_wslogging.xsl, ?) where this attribute should be used.

Comment (gaojx) [inline initfiles/componentfiles/configxml/mysqlserver.xsd]: The use is in the LN repository where the logging service is configured with mysql server. Will be in different ticket and pull request.

Comment (richardkchapman) [inline initfiles/componentfiles/configxml/mysqlserver.xsd]: The idea here is that you can specify a hostname without having to set up a machine in the environment and reference it... but existing components like eclserver referencing a legacy repository will also be affected.

I'll need to see the corresponding LN PR before I can comment on whether this change is safe/correct...

**Comment by gaojx** [comment]
We create a new one that combine all the changes together.

**Pull request closed without merging**